docs/cookbook.rst

```diff
@@ -50,15 +57,93 @@ adding a ``crispy`` template filter to the form. For example:
     <form action="{% url 'envelope-contact' %}" method="post">
         {% csrf_token %}
         {% antispam_fields %}
-        {{ form|crispy }}
-        <button type="submit">Send!</button>
+        {{ crispy form }}
```

Comment:
The correct template tag syntax is `{% crispy form %}` (guess that was just a typo). This means the sentence which introduces the code sample must be also updated to say *template tag* instead of *filter*.